docs/releases/3.2.txt

```diff
@@ -304,6 +304,10 @@ Tests
   :py:mod:`faulthandler` by default. This can be disabled by using the
   :option:`test --no-faulthandler` option.
 
+* Django now tracks timings like database creation, database teardown,
+  database cloning, total run time, and N slowest tests. This can be enabled by
+  using the :option:`test --timing` option. 
+
```

Comment:
This isn't right. The option isn't exposed to the `test` command. 

```
(master)$ ./manage.py test --timing
usage: manage.py test [-h] [--noinput] [--failfast] [--testrunner TESTRUNNER]
                      [-t TOP_LEVEL] [-p PATTERN] [--keepdb] [-r]
                      [--debug-mode] [-d] [--parallel [N]] [--tag TAGS]
                      [--exclude-tag EXCLUDE_TAGS] [--pdb] [-b]
                      [--no-faulthandler] [-k TEST_NAME_PATTERNS] [--version]
                      [-v {0,1,2,3}] [--settings SETTINGS]
                      [--pythonpath PYTHONPATH] [--traceback] [--no-color]
                      [--force-color]
                      [test_label [test_label ...]]
manage.py test: error: unrecognized arguments: --timing
```

We're not adding `N` here currently right? 

If this is just for `runtests.py` I'm not sure we need to doc it...

Reply:
I think that given that we're here already, exposing this to `django-admin test` is worthwhile, so argument, docs, and tests for that.

Reply:
But let's save `N` for another day.

Reply:
Agreed